docs/components/concepts/process-instance-modification.md

```diff
@@ -0,0 +1,226 @@
+
+Use the [modification command](/docs/apis-clients/grpc.md#modifyprocessinstance-rpc) to skip or repeat a step in the
+process. Consider the following example:
+
+![The process instance is stuck in the message catch event.](assets/process-instance-modification/process-instance-modification-example-1.png)
+
+The process contains two service tasks and a message catch event in between. The process instance completed the first
+task `A` and waits on the message catch event `B`. An external system should publish the message, but the external
+system is not available and can't continue the process. The process instance is stuck.
+
+![We use the modification to skip the event and continue on the next task.](assets/process-instance-modification/process-instance-modification-example-2.png)
+
+We use the modification to repair the process instance. We "move the token" from the catch event `B` to the next
+task `C`. This operation is presented by two instructions in the modification command:
+
+- Terminate the instance of the catch event `B`.
+- Activate the element `C`.
+
+![After the modification is applied, the message catch event is terminated and the next task is active.](assets/process-instance-modification/process-instance-modification-example-3.png)
+
+As a result of the command, the process instances terminated the instance of catch event `B` and activated the task `C`.
+Now, the process instance is not stuck anymore and can continue in the process.
+
+Generally, the process instance modification command can contain multiple instructions:
+
+- To activate an element of the process.
+- To terminate an active instance of an element.
+
+Read more about the behavior of the instructions in the following sections.
+
+:::note
+Use the process instance modification only in exceptional cases to repair the process instance. It is not
+recommended using it as a part of the regular flow of the process; find additional details [here](#use-at-your-own-risk).
+Instead, model all possible cases
+explicitly in your process.
+:::
+
+## Activate an element
+
+We can use the modification command to activate an element of the process. Consider the following example:
+
+![The process instance waits on task. We use the modification command to activate a task from a parallel flow.](assets/process-instance-modification/process-instance-modification-activate-an-element.png)
+
+The process instance completed the first task `A` and waits on task `B`. Task `C` is connected to task `A` by a
+non-interrupting message catch event. An external system should publish the message, but it is not available.
+
+To correct the state of the process instance, we modify it and activate task `C`. As a result, task `C` is active, and a
+job worker can pick it up.
+
+The process instance activates the element in the same way as the regular flow; for example, if the incoming sequence
+flow of the element would be taken. The activation of the element can include the following steps:
+
```

Comment:
Are the tabs here steps to complete in order? If so, I would instead list this in numerical order:
1. Apply the input variable mappings.
2. Create the event subscriptions; for example, of boundary events.
3. Apply additional logic depending on the element; for example, create a job for a service task.

Reply:
No. The order of the steps is not important for the docs. :+1: